Applications/ReferenceServer/README.md

```diff
@@ -1,12 +1,12 @@
 # OPC Foundation UA .Net Standard Library Reference Server
 
 ## Introduction
-This OPC Server is designed to be the default OPC UA Server when opening the [OPC UA Compliance Test Tool](https://opcfoundation.org/developer-tools/certification-test-tools/opc-ua-compliance-test-tool-uactt/) and it uses an address-space that matches the design of the UACTT. 
+This OPC Server is designed to be the default OPC UA Server when opening the [OPC UA Compliance Test Tool](https://opcfoundation.org/developer-tools/certification-test-tools/opc-ua-compliance-test-tool-uactt/) and it uses an address-space that matches the design of the UACTT and the requirements for OPC UA compliance testing. 
 
-It is built with the Quickstart application template (OPC Server) and uses the OPC Foundation UA .NET Standard Library SDK. Therefore it supports both the opc.tcp and https transports. There is a .Net 4.6 based server with UI and a .Net Standard 2.0 console version of the server which runs on any OS supporting [.NET Standard](https://docs.microsoft.com/en-us/dotnet/articles/standard).
+It uses the OPC Foundation UA .NET Standard Library. Therefore it supports both the opc.tcp and https transports. There is a .Net 4.6 based server with UI and a .Net Standard 2.0 console version of the server which runs on any OS supporting [.NET Standard](https://docs.microsoft.com/en-us/dotnet/articles/standard).
 
 ## How to build and run the Windows OPC UA Reference Server with UACTT
-1. Open the solution **UA Reference.sln** with Visual Studio 2017.
+1. Open the solution **UA-NetStandard.sln** with Visual Studio 2017.
```

Comment:
we should abandon the rpoject name UA-NetStandard and keep the UA Reference.sln

Reply:
Right.